bin/deploy_bosh

```diff
@@ -33,6 +33,13 @@ main() {
   bosh_env=$(cd "${1}"; pwd)
 
   iaas=$(bosh-cli int "${bosh_env}/director.yml" --path /iaas)
+  deploy_diagnostics=$(bosh-cli int ${bosh_env}/director.yml --path /deploy_diagnostics)
+
+  CREATE_ENV_SUFFIX=""
+  if [ "true" == "$deploy_diagnostics" ]; then
+    CREATE_ENV_SUFFIX="--ops-file \"$(repo_directory)/manifests/turbulence/manifest.yml\""
+  fi
+  export CREATE_ENV_SUFFIX
```

Comment:
Our deploy_<iaas> calls accept positional params inconsistently; passing it in would result in ugly conditionals downstream, hence the export.